operator_ui/src/pages/Chains/ChainNodes.tsx

```diff
@@ -3,19 +3,17 @@ import { Card, Grid } from '@material-ui/core'
 import Content from 'components/Content'
 import React from 'react'
 import { ChainResource } from '../ChainsIndex/ChainsIndex'
-import ChainNodesList from '../NodesIndex/NodesList'
-import { NodeResource } from '../NodesIndex/NodesIndex'
+import ChainNodesList from './NodesList'
```

Comment:
Moved some Nodes Table components into the Chains page because it still requires it.

Will remove this once the Chains pages get refactored

Reply:
:+1: